### PR TITLE
Update `wca-states.md`.

### PR DESCRIPTION
**Pull request by lgarron**
- Update names:
  - Macedonia → North Macedonia
  - Swaziland → Eswatini
- Fix sorting order.

### CODE REVIEW

File: wca-states.md [hunk 1]
```diff
@@ -77,8 +79,8 @@ The WCA recognizes a total of 199 countries/regions:
 - (GR:_Europe) Greece
 - (GD:_North America) Grenada
 - (GT:_North America) Guatemala
-- (GN:_Africa) Guinea
 - (GW:_Africa) Guinea Bissau
+- (GN:_Africa) Guinea
```
Comment (AlbertoPdRF) [inline wca-states.md]: Shouldn't `Guinea` go above `Guinea Bissau`?

Comment (lgarron) [reply wca-states.md]: Fixed!

**Comment by lgarron** [comment]
@julesDesjardin, could you review? :-D

**Review by Claster — APPROVED** [review]

**Comment by lgarron** [comment]
@Claster, thanks!

**Pull request merged**